vlsi/Makefile

```diff
@@ -87,6 +87,107 @@ $(SRAM_CONF): $(SRAM_GENERATOR_CONF)
 	cd $(vlsi_dir) && $(HAMMER_EXEC) -e $(ENV_YML) $(foreach x,$(INPUT_CONFS) $(SRAM_GENERATOR_CONF), -p $(x)) --obj_dir $(build_dir) sram_generator
 	cd $(vlsi_dir) && cp output.json $@
 
+#########################################################################################
+# simulation input configuration
+#########################################################################################
+include $(base_dir)/vcs.mk
+SIM_CONF = $(OBJ_DIR)/sim-inputs.yml
+SIM_DEBUG_CONF = $(OBJ_DIR)/sim-debug-inputs.yml
+SIM_TIMING_CONF = $(OBJ_DIR)/sim-timing-inputs.yml
+
+include $(vlsi_dir)/sim.mk
+$(SIM_CONF): $(VLSI_RTL) $(HARNESS_FILE) $(HARNESS_SMEMS_FILE) $(sim_common_files)
+	mkdir -p $(dir $@)
+	echo "sim.inputs:" > $@
+	echo "  top_module: $(VLSI_TOP)" >> $@
+	echo "  input_files:" >> $@
+	for x in $(HARNESS_FILE) $(HARNESS_SMEMS_FILE); do \
+		echo '    - "'$$x'"' >> $@; \
+	done
+	echo "  input_files_meta: 'append'" >> $@
+	echo "  timescale: '1ns/10ps'" >> $@
+	echo "  options: [" >> $@
+	echo "    '$(RISCV)/lib/libfesvr.a'," >> $@
+	echo "    '+lint=all,noVCDE,noONGS,noUI'," >> $@
+	echo "    '-error=PCWM-L'," >> $@
+	echo "    '-quiet'," >> $@
+	echo "    '-q'," >> $@
+	echo "    '+rad'," >> $@
+	echo "    '+v2k'," >> $@
+	echo "    '+vcs+lic+wait'," >> $@
+	echo "    '+vc+list'," >> $@
+	echo "    '-f $(sim_common_files)'," >> $@
+	echo "    '-sverilog'," >> $@
+	echo "    '-debug_pp']" >> $@
+	echo "  options_meta: 'append'" >> $@
+	echo "  defines: [" >> $@
+	echo "    'CLOCK_PERIOD=1.0'," >> $@
+	echo "    'PRINTF_COND=$(TB).printf_cond'," >> $@
+	echo "    'STOP_COND=!$(TB).reset'," >> $@
+	echo "    'RANDOMIZE_MEM_INIT'," >> $@
+	echo "    'RANDOMIZE_REG_INIT'," >> $@
+	echo "    'RANDOMIZE_GARBAGE_ASSIGN'," >> $@
+	echo "    'RANDOMIZE_INVALID_ASSIGN']" >> $@
+	echo "  defines_meta: 'append'" >> $@
+	echo "  compiler_opts: [" >> $@
+	echo "    '-I$(RISCV)/include'," >> $@
```

Comment:
Is the `-CC "-I$(VCS_HOME)/include"` done by default?

Reply:
will double check.

Reply:
Fixed